Documentation/HowToGuides/CreatingADrawer/README.md

```diff
@@ -0,0 +1,142 @@
+# Creating A Drawer
+
+> * Level: Beginner
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+***************************************************************************************
```

Comment:
A drawer is a simple example of how to use a Linear Drive to create a sliding object that is limited along an axis.